### PR TITLE
Feature/fixes for second release

### PR DESCRIPTION
**Pull request by StanislawSzataniak**
Commits are not preceeded with 'feature/...' becasue the naming script didn't work for me. Let me know if you see any mistakes.

### CODE REVIEW

**Comment by StanislawSzataniak** [comment]
Good news is naming script now works.

**Comment by StanislawSzataniak** [comment]
Automated branch naming

File: src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java [hunk 1]
```diff
@@ -9,61 +9,109 @@
...
         boolean levels = building.getLevels().size() == building.getLevels().stream().filter(distinctByKey(level -> level.getId())).count();
         boolean rooms = building.getLevels().stream()
             .allMatch(level -> level.getRooms().size() == level.getRooms().stream().filter(distinctByKey(room -> room.getId())).count());
         return levels && rooms;
     }
 
+    /**
+     * Function that returns whether given location is part of another
+     *
+     * @param l1 - location that contains the other location
```
Comment (grazik) [inline src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java]: I think that explanation of these params is not sufficent enough

File: src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java [hunk 2]
```diff
@@ -9,61 +9,109 @@
 import java.util.stream.Collectors;
 
 public class Janitor {
+
+    /**
+     * Function that combines all other verifying methods and returns whether the building structure is in order
+     * @param building
```
Comment (grazik) [inline src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java]: I think you may add some description here

File: src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java [hunk 3]
```diff
@@ -9,61 +9,109 @@
...
+     * @return true if building structure is correct, false otherwise
+     */
     public static boolean verifyBuildingStructure (Building building) {
         return verifyBuildingID(building) && verifyLevelsID(building) && verifyRoomsID(building) && verifyUnique(building);
     }
 
+    /**
+     * Function that returns whether the building's id is correct (matches the pattern b-0-0)
+     *
+     * @param building
```
Comment (grazik) [inline src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java]: I think you may add some description here

File: src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java [hunk 4]
```diff
@@ -9,61 +9,109 @@
...
+     */
     public static boolean verifyBuildingID (Building building) {
         String split[] = building.getId().split("-");
         return split.length == 3 && split[1].equals("0") && split[2].equals("0");
     }
 
+    /**
+     * Function that returns whether the levels' ids are correct (matches the pattern b-l-0)
+     *
+     * @param building
```
Comment (grazik) [inline src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java]: I think you may add some description here

File: src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java [hunk 5]
```diff
@@ -9,61 +9,109 @@
...
+     */
     public static boolean verifyLevelsID (Building building) {
         return building.getLevels().stream()
             .allMatch(level -> compareLocationID(building, level, 0) && level.getId().split("-")[2].equals("0"));
     }
 
+    /**
+     * Function that returns whether the rooms' ids are correct (matches the pattern b-l-r)
+     *
+     * @param building
```
Comment (grazik) [inline src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java]: I think you may add some description here

File: src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java [hunk 6]
```diff
@@ -9,61 +9,109 @@
...
     public static boolean verifyRoomsID (Building building) {
         return building.getLevels().stream()
             .allMatch(level -> level.getRooms().stream()
                 .allMatch(room -> compareLocationID(level, room, 0) && compareLocationID(level, room, 1)));
     }
 
+    /**
+     * Function that returns whether the ids in building are all unique
+     *
+     * @param building
```
Comment (grazik) [inline src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java]: I think you may add some description here

File: src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java [hunk 7]
```diff
@@ -9,61 +9,109 @@
...
         boolean rooms = building.getLevels().stream()
             .allMatch(level -> level.getRooms().size() == level.getRooms().stream().filter(distinctByKey(room -> room.getId())).count());
         return levels && rooms;
     }
 
+    /**
+     * Function that returns whether given location is part of another
+     *
+     * @param l1 - location that contains the other location
+     * @param l2 - location that is a part the other location
```
Comment (grazik) [inline src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java]: ` a part the other` 
I think you ate `of` ;)

**Comment by grazik** [comment]
> Automated branch naming, which means your script

Of course it does ^^
Why the first commits here are not named with our naming convention? :D

**Pull request merged**